sdk/nodejs/src/open-feature/DevCycleProvider.ts

```diff
@@ -250,9 +250,11 @@ export class DevCycleProvider implements Provider {
     ): ResolutionDetails<T> {
         return {
```

Comment:
Do you think we should add the other fields to the `metadata` param like Android? https://github.com/DevCycleHQ/android-client-sdk/pull/227#discussion_r2216603453